region/tourian/main.json

```diff
@@ -170,7 +170,11 @@
                       "excludedWeapons": [ "Super", "Missile" ]
                     }},
                     {"or": [
-                      {"metroidFrames": 300},
+                      {"enemyDamage": {
+                        "enemy": "Metroid",
+                        "type": "contact",
```

Comment:
I wonder if that attack would be clearer with a more descriptive name, like drain16Frames or something?

Reply:
I thought of "drain" myself but felt that maybe it's better to keep a contact value for all enemies. I'm not exactly strongly opinionated on this.